### PR TITLE
Added Become a Sponsor Page refs #53

### PR DESCRIPTION
**Pull request by danielmocan**

### CODE REVIEW

File: app/components/sponsors/becomeASponsor.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+      <strong>Interested in sponsoring JSHeroes?</strong>
+      <p>
+        Send us an email at <a href="mailto:contact@jsheroes.io">contact@jsheroes.io</a>. We will
+        follow-up and send you our current offer. Or you can download our{' '}
+        <a href="/static/brochure.pdf">Sponsorship Brochure</a> for more info.
+      </p>
+      <strong>What you get?</strong>
+      <ul>
+        <li>Direct access to 600+ developers</li>
```
[the remark this answers is not in the document]
Comment (danielmocan) [reply app/components/sponsors/becomeASponsor.js]: We should bundle all the css in one file, and create special .scss files for buttons, links, etc ... 18 px sounds good.

File: app/components/footer.js [hunk 2]
```diff
@@ -23,6 +23,11 @@ const Footer = () => (
             Buy Your Ticket
           </a>
         </div>
+        <div>
```
[the remark this answers is not in the document]
Comment (danielmocan) [reply app/components/footer.js]: Done

File: app/components/sponsors/becomeASponsorButton.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+const BecomeASponsorButton = () => (
+  <div className="button-wrapper">
+    <Style />
+    <a className="button" href="/become-a-sponsor">
```
Comment (alexnm) [inline app/components/sponsors/becomeASponsorButton.js]: use Link to avoid page reload

Comment (danielmocan) [reply app/components/sponsors/becomeASponsorButton.js]: done

**Pull request merged**